pkg/proxy/handlers/spacelister_get_test.go

```diff
@@ -726,3 +742,234 @@ func TestGetUserWorkspace(t *testing.T) {
 		})
 	}
 }
+
+func TestSpaceListerGetPublicViewerEnabled(t *testing.T) {
+
+	fakeSignupService := fake.NewSignupService(
+		newSignup("batman", "batman.space", true),
+		newSignup("robin", "robin.space", true),
+	)
+
+	fakeClient := fake.InitClient(t,
+		// space
+		fake.NewSpace("robin", "member-1", "robin"),
+		fake.NewSpace("batman", "member-1", "batman"),
+
+		// spacebindings
+		fake.NewSpaceBinding("robin-1", "robin", "robin", "admin"),
+		fake.NewSpaceBinding("batman-1", "batman", "batman", "admin"),
+
+		fake.NewSpaceBinding("community-robin", toolchainv1alpha1.KubesawAuthenticatedUsername, "robin", "viewer"),
+	)
+
+	robinWS := workspaceFor(t, fakeClient, "robin", "admin", true)
+	batmanWS := workspaceFor(t, fakeClient, "batman", "admin", true)
+	publicRobinWS := func() *toolchainv1alpha1.Workspace {
+		batmansRobinWS := robinWS.DeepCopy()
+		batmansRobinWS.Status.Type = ""
+		batmansRobinWS.Status.Role = "viewer"
+		return batmansRobinWS
+	}()
+	tests := map[string]struct {
+		username          string
+		expectedErr       string
```

Comment:
`expectedErr` is not used

Reply:
done in af7bb99eb78a40f3e966581e5038f715019d7f26